src/pages/dashboard/ui/ticket/TicketOptionPage.tsx

```
@@ -0,0 +1,146 @@
+              ...column,
+              taskIds: newTaskIds,
+            },
+          },
+        }));
+      }
+      return;
+    }
+
+    // options에서 ticket으로 이동할 때
+    if (source.droppableId === 'options' && destination.droppableId === 'ticket') {
+      const sourceColumn = data.columns[source.droppableId];
```

ESLint

[error] 'sourceColumn' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)